javascript/javascript2/week3/Exercises/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1></h1>
+    <button id="fButton">first</button>
+    <button id="sButton">second</button>

```

Comment:
firstButton(I hope you mean first button) is more specific than fButton. Try to give class names ,id, variable names , functions names to be specific, so that it will tell others what you indent to do